CHANGELOG.md

```diff
@@ -62,9 +62,9 @@ Ref: https://keepachangelog.com/en/1.0.0/
 ### Improvements
 
 * (x/auth) [\#8522](https://github.com/cosmos/cosmos-sdk/pull/8522) Allow to query all stored accounts
+* (x/ibc) [\#7949](https://github.com/cosmos/cosmos-sdk/issues/7949) Message `Acknowledgement` has been moved from channel.proto to acknowledgement.proto
```

Comment:
Should we also move the `PacketState` struct out of channel.

Open to either moving to `packet.proto` or putting `PacketState` and `Acknowledgement` structs into one separate proto file since they're conceptually linked

Reply:
This seems like a question for the cosmos/ibc repo